deploy/sitemap.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <urlset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9 http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd" xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
 <url>
-<loc>https://www.ergodox.io/</loc>
+<loc>http://0.0.0.0:3000/</loc>
```

Comment:
This shouldn't be 0.0.0.0:3000, I think this will be automatically fixed during the jekyll build but I will verify.